web/profiles/custom/os2loop/themes/os2loop_theme/templates/field/user-info.html.twig

```diff
@@ -0,0 +1,13 @@
+{#
+/**
+ * @file
+ * Implementation of user info related to content.
+ *
+ * Available variables:
+ * - user: The user to display info about.
+ */
+#}
+<div class="user">
+  {% include '@os2loop_theme/field/user-image.html.twig' with {'uid': user['#user'].id} %}
+  {% include '@os2loop_theme/field/user-name.html.twig' with {'user': user} %}
```

Comment:
If these two (small) templates are not used anywhere else I think we should just put their content into this template. If inlining is not possible, we should pass the full `user` object to `user-image.html.twig` for consistency.

Reply:
Combined into one